src/IECorePython/LRUCacheBinding.cpp

```diff
@@ -280,4 +353,7 @@ void IECorePython::bindLRUCache()
 		)
 	);
 
+	def( "testSerialLRUCacheRecursion", testSerialLRUCacheRecursion );
```

Comment:
We're currently using two unit test frameworks in Cortex (boost's and python's), and it's a real pain to maintain. In Gaffer we're only using Python's and binding any additional tests we want from the C++ side, and it's much better. My intention is to do the same for Cortex when I have the time. I think the LRUCache tests were originally bound this way because it made the performance testing much easier to because I could write ad-hoc scripts to drive it however was convenient at the time.